README.md

```diff
@@ -1 +1,8 @@
-# TF_wavelet
\ No newline at end of file
+# TF_wavelet
+
+
+## Authors 
+
+- Léa Latteur 
+- Edouard Delaire
+
```

Comment:
lease add Shahla, Christophe, and Jean-Marc

Reply:
done